### ISSUE TITLE
Apss not able to auto verify sms codes

### ISSUE THREAD

**Issue by W1zardK1ng** [GrapheneOS/os-issue-tracker#4549]
After resetting my phone im not able to login to some apps which auto verify sms codes sent to the mobile, im getting the text messages but the apps are failing to read it, there is no option to enter these codes manually. Trying in owner profile with google play installed and all the required permissions are granted also Exploit protection compatibility mode is enabled.

Apps: [PhonePe](https://play.google.com/store/apps/details?id=com.phonepe.app), [BHIM](https://play.google.com/store/apps/details?id=in.org.npci.upiapp&hl=en-US)
Device: Pixel 6a
Release: 2024121200

**Comment by W1zardK1ng** [comment]
Yes, SMS permission is granted to Google Play services also to the mentioned apps. Do you think adding any other permission to the play services will fix the issue?

**Comment by W1zardK1ng** [comment]
No but i added it now. Also they blocked me from trying to login to the apps for next 24 hrs due to multiple failed logins. ill update the status tomorrow.

**Comment by thereadonly** [comment]
Recent BHIM UPI, Phonepe and other UPI apps now require presence of Google Play services with the SMS permissions. This was not the case with BHIM UPI version 3.8.6 and below where the app directly read from the SMS

Relevant Discussions: 
https://discuss.grapheneos.org/d/21876-bhim-upi-stopped-working-after-updating
https://discuss.grapheneos.org/d/21899-upi-apps-not-auto-reading-sms-for-registration

This issue can be closed if there is no update from author.
